### ISSUE TITLE
About page: remove the "people" partial?

### ISSUE THREAD

**Issue by michalfrankl** [EHRI/ehri-nn-hugo-theme#3]
Since we also have the Contacts page, shouldn't we remove it here? I'm against unnecessary duplication.

**Comment by mikesname** [comment]
Fair point, though the design is different? 

Note that you can remove the people bit from the about page _content_ just by removing `type: about` from the frontmatter. This makes it use the standard template.

**Comment by michalfrankl** [comment]
Yes, removing or changing the type is indeed an easy solution. But to me, it's better not to duplicate, unless there is somewhat different information. My sense is that we better use the same layout for the same information.

But we can also wait and see what others think.
